formspree/templates/email/overlimit-notification.txt

```diff
@@ -2,7 +2,7 @@ Hey there,
 
 Someone just submitted your form on {{host}}.
 
-You are past our limit of {{ config.MONTHLY_SUBMISSIONS_LIMIT }} submissions per month, but the contents of this form were stored at our databases. You can access them by just logging in at {{ config.SERVICE_URL }}/register and upgrading your account.
+You are past our limit of {{ limit }} submissions per month, but the contents of this form were stored at our databases. You can access them by just logging in at {{ config.SERVICE_URL }}/register and upgrading your account.
```

Comment:
"You are past the limit of {{ limit }} free submissions per month. Never fear! We have safely stored the content of this form in our archive. You can access it by logging in at {{ url_for('register') }} and upgrading your account."